src/PointwiseFunctions/GeneralRelativity/Tags.hpp

```diff
@@ -177,6 +192,16 @@ template <typename DataType>
 struct WeylElectricScalar : db::SimpleTag {
   using type = Scalar<DataType>;
 };
+
+/*!
+ * \brief Computes the scalar \f$B_{ij} B^{ij}\f$ from the magnetic part of the
```

Comment:
change to `\brief The square \f$B_{ij} B^{ij}\f$ of the magnetic part of the Weyl tensor \f$B_{ij}\f$.`